webpush_channels/tests/test_views_channel.py

```diff
@@ -20,6 +23,28 @@ def test_registration_can_be_remove_for_a_user(self):
         resp = self.app.get(self.channel_url, headers=self.headers, status=200)
         assert resp.json['data']['registrations'] == 0
 
+    def test_remove_non_existent_registration(self):
+        self.assertRaises(self.app.delete(self.channel_registration_url,
```

Comment:
Why is that so? It returns a 500, though.
Shouldn't it be raising `kinto.core.storage.exceptions.RecordNotFoundError`?